NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(get_parameters)
+export(run_multiple_models)
```

Comment:
As these are *replicates* and not different models, perhaps we could go with `run_simulation_replicate`?